### PR TITLE
Increase branch coverage

### PR DESCRIPTION
**Pull request by jluech**
increasing branch coverage +25% (at least) for each touched unit

### CODE REVIEW

File: tests/de/gurkenlabs/litiengine/util/ArrayUtilitiesTests.java [hunk 1]
```diff
@@ -95,4 +118,24 @@ public void testDistinct() {
...
     assertArrayEquals(new Integer[] { 1, 2, 3, 4, 5, 6 }, result);
   }
+
+  @Test
+  public void testContains() {
+    Object [] first = new Object[] { 1, 2, 3, 4, 5, null };
+    Object [] second = new Object[] {};
+    Object [] third = new Object[1];
+
+    assertEquals(true, ArrayUtilities.contains(first, 2));
```
Comment (nightm4re94) [inline tests/de/gurkenlabs/litiengine/util/ArrayUtilitiesTests.java:128]: When checking for boolean conditions, use `assertTrue(...)` and `assertFalse(...)`. 
Please make sure this is done for all Tests that you introduced.

File: tests/de/gurkenlabs/litiengine/environment/MapObjectLoaderTests.java [hunk 2]
```diff
@@ -393,4 +393,16 @@ public int getBar() {
...
     }
   }
+
+  @Test
+  public void testMapObjectType(){
+
+    String mapObject1 = null;
+    String mapObject2 = "";
+
+    assertEquals(null, MapObjectType.get(mapObject1));
```
Comment (nightm4re94) [inline tests/de/gurkenlabs/litiengine/environment/MapObjectLoaderTests.java:403]: Use `assertNull(...)` here and make sure that all instances of this are fixed in the PR.

**Review by nightm4re94 — CHANGES_REQUESTED** [review]
Thanks a lot for your PR!
Everything looks good, I just noticed that in some instances, you could use `assertTrue()`, `assertFalse()`, and `assertNull()` instead of asserting equality. I have marked only two examples, but there were more than that. 
Please use the correct assertion methods and then I'll be happy to merge :)

**Comment by niels89** [comment]
@nightm4re94 Thanks for your review! I went through all of the added test cases and adjusted the assert methods, so that now the proper assert methods are used.

**Review by nightm4re94 — CHANGES_REQUESTED** [review]
Thank you for applying these changes so quickly!
I have one last request:
Your fixes have now introduced a bunch of wildcard imports - please replace them by explicit imports.

**Comment by niels89** [comment]
> 
> 
> Thank you for applying these changes so quickly!
> I have one last request:
> Your fixes have now introduced a bunch of wildcard imports - please replace them by explicit imports.

Done :)

**Review by nightm4re94 — APPROVED** [review]
thanks for this valuable contribution :)

**Pull request merged**